modules/sort/main/index.coffee

```diff
@@ -41,20 +41,16 @@ localeCollatorFn = (locale, options) ->
 # slower than compare but enforces locale comparison for browsers that
 # dont support Intl.Collator.
 localeCompare = (locale, options) ->
```

Comment:
Would it be sensible to use `options = {}` as the argument? Probably sensible to always merge into an object (essentially equivalent to having `options ?= {}` as the first line of the function